tables/he-IL-comp8.utb

```diff
@@ -31,54 +31,101 @@
 #  <http://www.gnu.org/licenses/>.
 #
 #-license: LGPLv2.1
-#-author-name: BAUM Engineering
-#-author-email: liblouis@baum.ro
+#-author-name: Adi Kushnir
+#-author-email: adi@accessmind.net
+#-author-name: Pavel Kaplan
+#-author-email: pavel46@gmail.com
 #-author-name: Erez Kugler
 #-author-email: erez@tsr-gaash.com
```

Comment:
Should one of these be the maintainer, the main contact person?

Reply:
I didn't even know about the `maintainer-name` and `maintainer-email meta data tag. I can't find any mention of it on [the wiki page](https://github.com/liblouis/liblouis/wiki/Table-discovery-based-on-table-metadata#standard-metadata-tags). I guess we could mention Adi Kushnir but we should ask him first

Reply:
Changed wiki page.